### ISSUE TITLE
Use cases for a new module/bundle to use the TornadoVM optionally?

### ISSUE THREAD

**Issue by JohannesLichtenberger** [sirixdb/sirix#610]
Maybe we can find use cases to leverage the TornadoVM :-)

**Comment by sardorjumamuratov** [comment]
Hi guys. I would like to work on this issue. Is this still available? 

Can I get the details of this issue?

**Comment by JohannesLichtenberger** [comment]
When I added the issue, I stumbled upon a YouTube video regarding TornadoVM and thought we might be able to leverage it in the future, but we haven't had a specific use case in mind. So it's up for research, and I'm all ears for ideas :-)

That said it should be optional :-)

**Comment by JohannesLichtenberger** [comment]
Still up for the task?

**Comment by sardorjumamuratov** [comment]
Yeah. I am a beginner though, I might need a little guidance. Thank you for that!
